stage1/usr_from_kvm/keys/lkvm

```diff
@@ -0,0 +1,12 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Does it mean that anyone (even non-root) who has IP connectivity to the container port 22 will be able to ssh using this well-known "private" key?